operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -384,6 +387,55 @@ function checkDirNotExistsOrEmpty() {
   trace "Exiting checkDirNotExistsOrEmpty"
 }
 
+# limit the file extensions in the model directories
+
+function checkModelDirectoryExtensions() {
+  trace "Entering checkModelDirectoryExtensions"
+
+  cd ${IMG_MODELS_HOME}
+  counter=$(ls  -I  *.yaml -I *.zip -I *.properties | wc -l)
+  if [ $counter -ne 0 ] ; then
+        trace SEVERE "Directory ${IMG_MODELS_HOME} contains files with unsupported extension. Extension must be" \
```

Comment:
extension -> extensions

Reply:
Extension must be -> Expected extensions

Reply:
done